.eslintignore

```diff
@@ -0,0 +1 @@
+/addon/build
```

Comment:
Ignoring the directory (same like we do in .gitignore) since it currently only contains the ./addon/build/testpilot-ga.js file that gets copied over from ./node_modules/ during the <kbd>npm run build</kbd> step.